tests/test_distributed_model.py

```diff
@@ -49,3 +49,7 @@ def test_distributed_model_prefill_gpt2():
 @pytest.mark.slow
 def test_distributed_model_prefill_gemma7b():
     _test_distributed_model_prefill("google/gemma-7b")
+
+@pytest.mark.slow
+def test_distributed_model_prefill_llama3_8b():
+    _test_distributed_model_prefill("meta-llama/Meta-Llama-3-8B")
```

Comment:
Should we start parametrize the test rather than dupplicate?

Reply:
I should definitely do a bit of refactoring on the tests to avoid duplicaiton... will do later if you agree.